cluster-up/cluster/kind/manifests/kube-calico.diff.in

```diff
@@ -1,5 +1,5 @@
---- kube-calico.yaml.in	2021-03-18 09:52:16.000000000 +0200
```

Comment:
Leftovers from a rebase that went not correct, please check it

Reply:
Removed.